### PR TITLE
Create and tested /api/articles/:article_id/comments route

### PR DESCRIPTION
**Pull request by MarKo-ctrl**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -226,4 +226,63 @@ describe('GET /api/articles', () => {
...
+                            votes: expect.any(Number),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            created_at: expect.any(String),
+                        }));
+                });
+            });
+    });
+
+    it('404: responds with a not found message when there are no comments for the given article', () => {
```
Comment (k-huggs) [inline __tests__/app.test.js:253]: If there are no comments for an article, that wouldn't necessarily be an error. All new artiles will start with no comments until someone post a comment.

**Review by k-huggs — COMMENTED** [review]
Hi, slight alteration to be made!

File: __tests__/app.test.js (hunk 1 above)
Comment (MarKo-ctrl) [reply __tests__/app.test.js]: Fixed

File: __tests__/app.test.js [hunk 2]
```diff
@@ -226,4 +226,63 @@ describe('GET /api/articles', () => {
...
+        const ARTICLE_ID = 'VII';
+        return request(app)
+            .get(`/api/articles/${ARTICLE_ID}/comments`)
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Invalid ID');
+            });
+    });
+
+    it('404: responds with a not found message when the route passed does not exist', () => {
```
Comment (DCrawley94) [inline __tests__/app.test.js:280]: This test doesn't really fit into this describe block. It's testing the default express behaviour when a route isn't found and the describe block specifies the endpoint should match `GET /api/articles/:article_id/comments`

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
All looking good Marios. Thought I'd suggest removing the final test as it doesn't really fit the description for these tests

Once it moved/removed feel free to merge :)

File: __tests__/app.test.js (hunk 2 above)
Comment (MarKo-ctrl) [reply __tests__/app.test.js]: Fixed

**Pull request merged**